misc/gen_types.py

```diff
@@ -13,24 +13,26 @@ def {name}(self) -> {type}: ..."""
 
 
 def replace_type(typename):
+    KEYWORD_REPLACEMENT = [("Bytes", "bytes"), ("Float", "float")]
+    for kw, value in KEYWORD_REPLACEMENT:
+        if typename == kw:
+            return value
+
+
+def replace__wrapped_type(typename):
```

Comment:
```suggestion
def replace_wrapped_type(typename):
```
![image](https://user-images.githubusercontent.com/30697622/190401267-6a5a6608-be33-46d4-8fa9-d492c89bf9c1.png)